tests/conftest.py

```diff
@@ -455,280 +305,125 @@ def dummy_climate_data(fixture_core_components):
-        np.broadcast_to(absorbed, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="absorbed_radiation",
-        coords=full_coordinates,
-    )
-    data["sensible_heat_flux_topofcanopy"] = DataArray([100, 50, 10], dims=["cell_id"])
-    data["sensible_heat_flux_soil"] = DataArray([1, 1, 1], dims=["cell_id"])
-    data["latent_heat_flux_soil"] = DataArray([1, 1, 1], dims=["cell_id"])
-    data["friction_velocity"] = DataArray([12, 5, 2], dims=["cell_id"])
-    sensible_heat_flux = np.repeat(a=[0.0, np.nan, 0.0, np.nan], repeats=[4, 9, 1, 1])
-    data["sensible_heat_flux"] = DataArray(
-        np.broadcast_to(sensible_heat_flux, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="sensible_heat_flux",
-        coords=full_coordinates,
-    )
-    latent_heat_flux = np.repeat(a=[0.0, np.nan, 0.0, np.nan], repeats=[4, 9, 1, 1])
-    data["latent_heat_flux"] = DataArray(
-        np.broadcast_to(latent_heat_flux, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="latent_heat_flux",
-        coords=full_coordinates,
-    )
-    molar_density_air = np.repeat(a=[38.0, np.nan, 38.0, np.nan], repeats=[4, 7, 2, 2])
-    data["molar_density_air"] = DataArray(
-        np.broadcast_to(molar_density_air, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="molar_density_air",
-        coords=full_coordinates,
-    )
-    specific_heat_air = np.repeat(a=[29.0, np.nan, 29.0, np.nan], repeats=[4, 7, 2, 2])
-    data["specific_heat_air"] = DataArray(
-        np.broadcast_to(specific_heat_air, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="specific_heat_air",
-        coords=full_coordinates,
-    )
-    data["zero_displacement_height"] = DataArray(np.repeat(20.0, 3), dims="cell_id")
-    data["diabatic_correction_heat_above"] = DataArray(
-        np.repeat(0.1, 3),
-        dims=["cell_id"],
-        name="diabatic_correction_heat_above",
-    )
-    data["diabatic_correction_heat_canopy"] = DataArray(
-        np.repeat(1.0, 3),
-        dims=["cell_id"],
-        name="diabatic_correction_heat_canopy",
-    )
-    data["diabatic_correction_momentum_above"] = DataArray(
-        np.repeat(0.1, 3),
-        dims=["cell_id"],
-        name="diabatic_correction_momentum_above",
-    )
-    data["diabatic_correction_momentum_canopy"] = DataArray(
-        np.repeat(1.0, 3),
-        dims=["cell_id"],
-        name="diabatic_correction_momentum_canopy",
-    )
-    attenuation_coefficient = np.repeat(
-        a=[13.0, np.nan, 2.0, np.nan], repeats=[4, 7, 2, 2]
-    )
-    data["attenuation_coefficient"] = DataArray(
-        np.broadcast_to(attenuation_coefficient, (3, 15)).T,
-        dims=["layers", "cell_id"],
-        name="attenuation_coefficient",
-        coords=full_coordinates,
-    )
-    data["mean_mixing_length"] = DataArray(np.repeat(1.3, 3), dims="cell_id")
+    data["absorbed_radiation"] = from_template()
+    data["absorbed_radiation"][[1, 2, 3]] = 10.0
```

Comment:
I think this variable is no longer used, we have the `canopy_absorption` instead